spec/features/user_signs_up_spec.rb

```diff
@@ -15,6 +15,7 @@
   scenario 'provide valid registration information' do
     visit new_user_registration_path
 
+    fill_in 'Name', with: 'Charlotte'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.